pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -93,6 +94,12 @@ stdenv.mkDerivation rec {
 
     echo "=== Running MathInstaller ==="
     ./MathInstaller -auto -createdir=y -execdir=$out/bin -targetdir=$out/libexec/Mathematica -silent
+
+    # Fix library paths
```

Comment:
Note that we have zlib 1.2.11 currently and not 1.2.9. I could imagine issues with using our version instead of the vendored one. From your list it seems pango is vendored already as well.